app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/websocket/WebSocketServerService.kt

```diff
@@ -3,14 +3,11 @@ package co.netguru.baby.monitor.client.feature.communication.websocket
 import android.app.Service
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/59ecd15255bdd529a391ea0d5072dc37d88e9138/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/websocket/WebSocketServerService.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>